src/main/java/com/adaptris/filesystem/FileExtractionMode.java

```diff
@@ -41,13 +43,17 @@
 
   private transient Logger log = LoggerFactory.getLogger(this.getClass().getName());
 
-  private MessageDrivenDestination filenameMatch;
+  @Getter
+  @Setter
+  @NotBlank
+  @InputFieldHint(expression = true)
+  private String filenameMatch;
 

```

Comment:
Don't think filenameMatch should be annotated with NotBlank since later on you have

```
if (getfilenameMatch() != null) {
}
```

Please add some documetnation as to what the filenameMatch does as well .